<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmXProgressBar</class>
 <widget class="QWidget" name="frmXProgressBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="XProgressBar" name="progressPlay1" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="XProgressBar" name="progressPlay2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="XProgressBar" name="progressPlay3" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="XProgressBar" name="progressPlay4" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>220</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="labValue">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>当前进度: 0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QSlider" name="horizontalSlider">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="XProgressBar" name="progressPlay6" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="XProgressBar" name="progressPlay5" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>145</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XProgressBar</class>
   <extends>QWidget</extends>
   <header>xprogressbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
